feed/obs/thread.js

```diff
@@ -19,20 +21,23 @@ exports.create = function (api) {
     if (!ref.isLink(rootId)) throw new Error('an id must be specified')
     var sync = Value(false)
 
+    const blocking = api.contact.obs.blocking(api.keys.sync.id())
+
     var prepend = MutantArray()
     api.sbot.async.get(rootId, (err, value) => {
       sync.set(true)
       if (!err) {
-        prepend.push(
-          Value(unboxIfNeeded({key: rootId, value}))
-        )
+        var msg = unboxIfNeeded({key: rootId, value})
```

Comment:
This will make navigating to posts by blocked people a little bit confusing, just a missing message. 

I think that If you explicitly navigate to a thread by ID and the author was blocked, the interface should probably warn you that you are viewing a post by a person that you have blocked, and tell you who that person is. 

Maybe the thread code should be updated to include a blocked flag in the observable so that this can be detected in patchwork/bay and rendered accordingly.

Reply:
Yeah ok I was actually testing that case. Updated code. I suggest you search for #nsfw for example of something to block :D

Reply:
If you link to a specific message you could just add a data-attribute or class to what's rendered. Then extend the default post rendering to allow you to remove the class if you really want to  : 


here's a mockup of what I was able to do with css to a pretty bad #nsfw post : 

![selection_405](https://user-images.githubusercontent.com/2665886/31051811-de7f96a2-a6ce-11e7-8fb9-a4d698a203d8.png)


```css
    filter: blur(40px) grayscale(100%);
    opacity: .5;
    max-height: 200px;
    overflow-y: hidden;
```